packages/hooks/tests/useNFTReference.test.ts

```diff
@@ -0,0 +1,41 @@
+import { enableFetchMocks } from 'jest-fetch-mock'
+enableFetchMocks()
+import { renderHook } from '@testing-library/react-hooks'
+import { useNFTReference } from '../src/NFT/useNFTReference'
+import { cache } from 'swr'
+import type { FetchMock } from 'jest-fetch-mock'
+const fetchMock = fetch as FetchMock
+
+describe('useNFTReference', () => {
+    beforeEach(() => {
+        cache.clear()
+    })
+    test('should return a Json from a URI', async () => {
+        fetchMock.mockResponseOnce(JSON.stringify({ data: '123' }))
```

Comment:
Is this actually being used here?

Reply:
Yeah, it fakes the fetch function response. The reason I used this, is that the `node-fetch` used by `jest-fetch-mock` handle the fetching differently than it's done using the browser `fetch` function. And If we don't mock the response, we'll get an inconsistent response from the browser.

As long as the fetch function is called and handled the response correctly, that means the hook is working and the test should pass.